doc/nrf/known_issues.rst

```diff
@@ -363,6 +363,19 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+DRGN-15435: GATT notifications and Writes Without Response might be sent out of order.
```

Comment:
```suggestion
DRGN-15435: GATT notifications and Writes Without Response might be sent out of order
```
Third attempt to get rid of that period ... ;)

Reply:
...

Reply:
Fingers crossed it is gone now :)